app/src/App.tsx

```diff
@@ -174,7 +176,10 @@ const AppWithRouter = (props) => {
     <Router>
       <ErrorDisplayBoundary>
         <CountProvider>
-          <App componentsData={componentsData} />
+          <Switch>
+            <Route path="/loggedin" exact component={LoginRedirectPage} />
+            <Route path="/" exact={false} component={App} componentsData={componentsData} />
```

Comment:
This doesn't actually pass the componentsData prop to the component unfortunately. Router doesn't seem to pass them 😞 

Here's an alternative you can do which does using render:
```suggestion
            <Route path="/" exact={false} render={passed_props => <App {...passed_props} componentsData={componentsData} />} />
```

Reply:
Thank you @shaunmaharaj